content/1.7/back-office/shop-parameters/search/search-listing.md

```diff
@@ -0,0 +1,179 @@
+The following tooltip is displayed: "Note that this option is resource-consuming: the more you search, the longer it takes."
+ - "Maximum word length (in characters)". This option defines how many characters you allow to run a fuzzy search query. This field is mandatory, when trying to fill it out when it is not filled out, shows the message: "field Maximum word length (in characters) is required." The field also can be filled only with characters, if any other symbol will be entered, when trying to save, will show the message: "The Maximum word length (in characters) field is invalid."
+An explanation below is shown: "This parameter will only be used if the fuzzy search is activated: the lower the value, the more tolerant your search will be.".
+Also a tooltip is displayed: "Only words fewer or equal to this maximum length will be searched."
+ - "Minimum word length (in characters)". It is possible to choose the minimum size at which a word may be registered in the search index and found the customers. Field only accepts characters, if any other symbol is input, when trying to save, shows the following message: "The Minimum word length (in characters) field is invalid." 
+Also has a tooltip: "Only words this size or larger will be indexed."
+ - "Blacklisted words". It allows to choose the terms that must not be found by the shop visitors. Has a tooltip: "Please enter the index words seperated by a".
+The field is multilanguage, can be translated to the needed languages which are in the shpo with a drop field on the right of the field.
+By default this field has input in the field the following: "a|about|above|after|again|against|all|am|an|and|any|are|aren|as|at|be|because|been|before|being|below|between|both|but|by|can|cannot|could|couldn|did|didn|do|does|doesn|doing|don|down|during|each|few|for|from|further|had|hadn|has|hasn|have|haven|having|he|ll|her|here|hers|herself|him|himself|his|how|ve|if|in|into|is|isn|it|its|itself|let|me|more|most|mustn|my|myself|no|nor|not|of|off|on|once|only|or|other|ought|our|ours|ourselves|out|over|own|same|shan|she|should|shouldn|so|some|such|than|that|the|their|theirs|them|themselves|then|there|these|they|re|this|those|through|to|too|under|until|up|very|was|wasn|we|were|weren|what|when|where|which|while|who|whom|why|with|won|would|wouldn|you|your|yours|yourself|yourselves"
+
+ # Weight
+ 
+ PrestaShop enables to prioritize certain data when a search is performed on the shop.
+ 
+ The following information is presented in this section:
+ 
+ "The "weight" represents its importance and relevance for the ranking of the products when completing a new search.
+A word with a weight of eight will have four times more value than a word with a weight of two.
+
+We advise you to set a greater weight for words which appear in the name or reference of a product. This will allow the search results to be as precise and relevant as possible.
```

Comment:
```suggestion
"We advise you to set a greater weight for words which appear in the name or reference of a product. This will allow the search results to be as precise and relevant as possible."
```